WRAPPERS.md

```diff
@@ -2,8 +2,7 @@
 | -----------------------------| -------|   ---- | ------------------------|
 | Specific python wrapper      | Yes    | Yes    | working (2021-01-21)    |
 | Openapi python wrapper       | No     | Yes    | proof of concept only   |
-| Bash wrapper                 | No     | Yes    | might be outdated       |
-| Java library                 | No     | Yes    | working (2021-04-13)    |
+| Java library                 | No     | Yes    | working (2021-08-30)    |
 | Image using the Java library | No     | Yes    | out of date             |
```

Comment:
Is this image really still out of date, when it depends on the Java library which is working?